build-system/release-tagger/test/make-release.test.js

```diff
@@ -1,6 +1,6 @@
 const nock = require('nock');
```

Comment:
I mentioned wanting to write / update a test in #37737. I'd like to get this through sooner rather than later though as publishing is currently broken.

Reply:
Is it broken or just missing the recently moved packages? We publish on a nightly basis and it's an easy workflow trigger to set one of the nightlies as latest.

Would like to see updated tests in this PR since it affects the GitHub release body. Should be straight forward but lmk if you need a pointer.